Hestia/app/src/main/java/hestia/backend/exceptions/ServerExceptions/SetupFailedException.java

```diff
@@ -0,0 +1,22 @@
+package hestia.backend.exceptions.ServerExceptions;
+
+import com.google.gson.JsonObject;
+
+public class SetupFailedException extends ServerException {
+    String field;
+    String hint;
+
+    public SetupFailedException(String error, JsonObject details){
```

Comment:
don't think i get what you mean, Gson is not used in the parsing of the informationn right?